src/OpenTelemetry/Logs/Builder/LoggerProviderBuilderBase.cs

```diff
@@ -23,20 +23,17 @@
 namespace OpenTelemetry.Logs;
 
 /// <summary>
-/// Contains methods for registering actions into an <see
-/// cref="IServiceCollection"/> which will be used to build a <see
-/// cref="LoggerProvider"/> once the <see cref="IServiceProvider"/> is
-/// available.
+/// Contains methods for building <see cref="LoggerProvider"/> instances.
 /// </summary>
-internal sealed class LoggerProviderServiceCollectionBuilder : LoggerProviderBuilder, ILoggerProviderBuilder
+internal sealed class LoggerProviderBuilderBase : LoggerProviderBuilder, ILoggerProviderBuilder
```

Comment:
@utpilla 

Originally when this was added you commented that you would prefer it was named `LoggerProviderBuilderBase` so it matched tracing & metrics. Just FYI it ended up being exactly that 😄